modules/next/tests/src/Kernel/Event/EntityActionEventTest.php

```diff
@@ -63,39 +62,36 @@ public function testEntityActionEvents() {
     // Insert.
     $page->save();
     $this->container->get('kernel')->terminate(Request::create('/'), new Response());
-    $this->assertLogsContains("Event next.entity.action dispatched for entity A page and action insert.");
+    $this->assertLogMessage("insert");
 
     // Update.
     $page->set('title', 'A page updated')->save();
     $this->container->get('kernel')->terminate(Request::create('/'), new Response());
-    $this->assertLogsContains("Event next.entity.action dispatched for entity A page updated and action update.");
-
-    // Delete.
-    $page->delete();
-    $this->container->get('kernel')->terminate(Request::create('/'), new Response());
-    $this->assertLogsContains("Event next.entity.action dispatched for entity A page updated and action delete.");
+    $this->assertLogMessage("update");
   }
 
   /**
-   * Helper to assert logs.
+   * Helper to assert log.
    *
-   * @param string $message
-   *   The message to assert in the logs.
+   * @param string $action
+   *   The action to perform.
    */
-  protected function assertLogsContains(string $message) {
-    $logs = Database::getConnection()
-      ->select('watchdog', 'wd')
-      ->fields('wd', ['message', 'variables'])
+  protected function assertLogMessage(string $action) {
+    $message = "Event @event dispatched for entity @label and action @action.";
+    $variables = [
+      '@event' => 'next.entity.action',
+      '@label' => 'A page',
+      '@action' => $action,
+    ];
+
+    $this->assertNotEmpty(Database::getConnection()->select('watchdog', 'w')
```

Comment:
@apathak18 I'm not sure on this, but we can use it like below instead of using `Database` class:
```suggestion
    $this->assertNotEmpty($this->container->get('database')->select('watchdog', 'w')
```